config/application.yml.example

```diff
@@ -127,6 +127,7 @@ development:
   state_id_proofing_vendor: 'mock'
   telephony_disabled: 'true'
   twilio_accounts: '[{"sid":"sid", "auth_token":"token", "number":"9999999999"}]'
```

Comment:
Should we remove the `twilio_accounts` entries if they will no longer be used?

Reply:
We still use these for phone numbers